src/packages/utils/src/utils/request-coordinator.ts

```diff
@@ -82,9 +115,15 @@ export class RequestCoordinator {
     thisArgument: any,
     argumentsList: OverloadedParameters<T>
   ) => {
+    if (this.#stopped) {
```

Comment:
I thought I left a comment about this yesterday, but I liked your [original approach](https://github.com/trufflesuite/ganache/pull/3433/commits/0638bf651a0302cbd88939f2b83c04bc0ab81be6#diff-de6272ed4a53f5d059f4c98d25691912a50df13d02ec23f909958cc9a7760ffeR83-R89) where you replaced `queue` and `resume` with functions that reject. Doing it this way made it so that the `this.#stopped` property wasn't unnecessarily checked on every request. What was the reason for changing it?

Reply:
I liked that for the same reason. I changed it however, because there could be events already scheduled referencing the original functions, which would end up throwing internal errors if they execute after `stop()` is called.

Alternatively, I could have added an _internal_ function that I switched out, but then I'm tossing up between a boolean value check vs an additional function call on the stack 🤷

Reply:
Ah. I didn't think about other code peeling off the methods and using them later. Do we do that anywhere currently?

I'm not sure we should be that concerned with the behavior of a function after it has been removed from its original call context.

```
const queue = requestCoordinator.queue.bind(requestCoordinator);
// then later...
queue() // <-- could we get away with calling this sort of pattern "undefined behavior"
```

Reply:
Yeah, especially because the function is called only in `Executor` which is basically "pass-through", I think it's fine. I liked the idea of making the interface as "correct" as possible, but maybe not worth it.

I'm going to add a comment explaining the edge case to ensure I don't forget and break it in a future change :P